### ISSUE TITLE
is cheat engine safe ?

### ISSUE THREAD

**Issue by s9be** [cheat-engine/cheat-engine#1448]
https://www.virustotal.com/gui/file/3c1ead7f3d812e2a3cf09c382ecc71212f5c86ce6a2da7f4c7033925241ad53d/detection

i use norton antivirus but it didn't detect anything i'm really worry can someone Enlighten me

**Comment by cheat-engine** [comment]
it runs as admin and lets you do whatever you like.

If you tell CE to delete all files on your system, the answer is yes

**Comment by Kellegram** [comment]
> 
> 
> https://www.virustotal.com/gui/file/3c1ead7f3d812e2a3cf09c382ecc71212f5c86ce6a2da7f4c7033925241ad53d/detection
> 
> i use norton antivirus but it didn't detect anything i'm really worry can someone Enlighten me
> 
> can it harm my pc ?

It installs malware if you have internet connection during install, you need to make sure you have no connection. They close all Issues raised by people about it without leaving a comment. Installing via scoop is safe tho.

**Comment by fade2gray** [comment]
@Kellegram Scoop forces you disable your browser's adblocker in order to access their website - what does that tell you about Scoop?

From the Scoop website...

> Your usual installation procedure might look something like this: navigate to a download website, download the installer and click through the prompts, **`hopefully avoiding any ads`** which might try to distract you during the procedure.

**Comment by Th3C0D3R** [comment]
It installs malware only if u rush through the installer without check what you click on or if you are just stupid 😂😂  first leads to the second one

**Comment by Kellegram** [comment]
Nope, that's wrong and has been proven wrong, opting out during installation doesn't save you from all of the malware, jesus, every single thread needs this repeated. The only way to avoid this is to disconnect from the internet during installation.

**Comment by Kellegram** [comment]
> 
> 
> @Kellegram Scoop forces you disable your browser's adblocker in order to access their website - what does that tell you about Scoop?
> 
> From the Scoop website...
> 
> > Your usual installation procedure might look something like this: navigate to a download website, download the installer and click through the prompts, **`hopefully avoiding any ads`** which might try to distract you during the procedure.

No idea what you are trying to say here, especially with the quote, the quote is not related to what you said nor is it important. I don't use scoop anyway, I just know scoop works around this issue.

**Comment by spidychoipro** [comment]
yeah i download the source code but it says 11 virus detected.

**Comment by fade2gray** [comment]
@spidychoipro Name the 11 **VIRUSES** detected.
